sdk/ai/azure-ai-projects/README.md

```diff
@@ -718,6 +718,122 @@ print("Deleted agent")
 
 <!-- END SNIPPET -->
 
+### Evaluation
+Evaluation in Azure AI Project client library is desinged to assess the performance of generative AI applications in cloud. Generative AI application generations are quantitatively measured with mathematical based metrics, AI-assisted quality and safety metrics. Metrics are defined as evaluators. Built-in or custom evaluators can provide comprehensive insights into the application's capabilities and limitations.
+
+#### Evaluator
+Evaluators are custom or prebuilt classes or functions that are designed to measure the quality of the outputs from language models or generative AI applications.
+
+Evaluators are made available via [azure-ai-evalaution][azure_ai_evaluation] SDK for local experience and also in [Evaluator Library][evaluator_library] in Azure AI Studio for using them in the cloud.
+
+More details on built-in and custom evaluators can be found [here][evaluators].
+
+#### Run Evalaution in cloud:
+
+To run evaluation in cloud following are needed:
+
+- Evaluators
+- Data to be evaluated
+- [Optional] Azure Open AI model.
+
```

Comment:
Can you say something about supporting non OpenAI models? Or is that too soon

Reply:
Right now we do not support that. Once we add support will make an update.